### DISCUSSION TITLE
Relationship and choice

### DISCUSSION THREAD

**Discussion by masksshow** [aminalaee/sqladmin#272] [Q&A]
Hi.
I am using sqladmin==0.2.1. 
My models have a relationship. I'm trying to edit or add records.
What do I do if I have a huge number of entries?
They all appear in the selection box.
 
![alt text](https://i.imgur.com/b9XsXlS.png)

**Comment by aminalaee** [comment]
Hey,
This is already started here. It needed a few internal improvements and is now in progress. It will be available soon.

https://github.com/aminalaee/sqladmin/issues/193

**Comment by aminalaee** [comment]
Merged in https://github.com/aminalaee/sqladmin/pull/292 and will be available in `0.5.0`.
